ci/cray/Jenkinsfile.verbs

```diff
@@ -0,0 +1,147 @@
+// Libfabric Verbs provider Jenkins Pipeline file
+// Copyright 2018. Cray Inc.
+
```

Comment:
Just a general comment: When you are ready for me to merge this pull request just let me know (or cray_test can do it). I don't have the ability (or Jenkins knowledge) to validate that it is correct. Thanks.